### DISCUSSION TITLE
Replace macOS app icon from terminal

### DISCUSSION THREAD

**Discussion by azat-io** [elrumo/macOS_Big_Sur_icons_replacements#1420] [General]
Hey. Is there any way to replace icon from terminal?

**Comment by vanyauhalin** [comment]
Hi. I'm developing [Fresco](https://github.com/vanyauhalin/Fresco), an icon manager, which already has CLI with base commands. There is still a lot of work ahead, but you can already start using it. Will be happy to see your issues or bug reports.

**Comment by dch82** [comment]
[https://www.sethvargo.com/replace-icons-osx](https://www.sethvargo.com/replace-icons-osx)

TL;DR: Check in `[App Name].app/Contents/Info.plist` for the entry under `CFBundleIconFile`.
Copy that, then run something like
```
cp /path/to/icon.icns /path/to/[App Name].app/Contents/Resources/[CFBundleIconFile]
```

Have a nice day!
